infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/DefaultClusterManager.java

```diff
@@ -112,9 +171,8 @@ public static TopologyDescriptor readConfig() throws IOException {
             }
             // TODO: add reading of node id (which is the APH node uuid)
 
-            reader.close();
-        } catch (FileNotFoundException e) {
-            log.warn("Plugin Config File {} does not exist. Using default configs", config_file);
+        } catch (IOException e) {
+            log.warn("Plugin Config File {} does not exist. Using default configs", CONFIG_FILE_PATH);
```

Comment:
this can be an inaccurate error message if the exception was not FileNotFound.  Why was the exception type changed?

Reply:
Auto close will throw IOException. I think it is fine, since the entire default cluster manager is test only.